### PR TITLE
Add note for password on BACKEND_HOST

### PR DESCRIPTION
**Pull request by gabi18**
Add the hint for account settings on BACKEND_HOST, see https://github.com/owncloud/client/issues/8841#issuecomment-889034155

### CODE REVIEW

File: modules/ROOT/pages/appendices/guitest.adoc [hunk 1]
```diff
@@ -80,6 +80,9 @@ Some necessary steps before running tests:
 . Clone the {client-repo-url}[Desktop Client] from GitHub
 . Copy `test/gui/config.sample.ini` to `test/suite_oc-desktop/config.ini`
 . Edit `test/gui/config.ini` and set `BACKEND_HOST=` to the URL of your ownCloud server, e.g. `BACKEND_HOST=http://localhost/owncloud-core`
+
+NOTE: BACKEND_HOST can be any server but it's required that password for user 'admin' is set to 'admin'
```
Comment (mmattel) [inline modules/ROOT/pages/appendices/guitest.adoc:84]:
```suggestion
NOTE: BACKEND_HOST can be any server, but it is required that the password for the user `admin` is set to `admin`.
```
Small text fix

Comment (mmattel) [inline modules/ROOT/pages/appendices/guitest.adoc]:
```suggestion
+
NOTE: BACKEND_HOST can be any server but it's required that password for user 'admin' is set to 'admin'
```
The `+` sign makes the following note part of the list item an does not break the list intent

File: modules/ROOT/pages/appendices/guitest.adoc [hunk 2]
```diff
@@ -80,6 +80,9 @@ Some necessary steps before running tests:
 . Clone the {client-repo-url}[Desktop Client] from GitHub
 . Copy `test/gui/config.sample.ini` to `test/suite_oc-desktop/config.ini`
 . Edit `test/gui/config.ini` and set `BACKEND_HOST=` to the URL of your ownCloud server, e.g. `BACKEND_HOST=http://localhost/owncloud-core`
+
+NOTE: BACKEND_HOST can be any server, but it is required that the password for the user `admin` is set to `admin`.
```
Comment (mmattel) [inline modules/ROOT/pages/appendices/guitest.adoc:84]:
```suggestion
+
NOTE: BACKEND_HOST can be any server, but it is required that the password for the user `admin` is set to `admin`.
```
The `+` sign makes the NOTE part of the list item and does not break the list intent

**Comment by mmattel** [comment]
@gabi18 is this for 2.9 only, or does it need to be backported to 2.8 too

**Comment by phil-davis** [comment]
> @gabi18 is this for 2.9 only, or does it need to be backported to 2.8 too

Automated tests started in 2.9 only. No backport needed.

**Review by phil-davis — APPROVED** [review]

**Comment by gabi18** [comment]
Thanks!

**Pull request merged**